pkg/ingester/flush.go

```diff
@@ -2,6 +2,7 @@ package ingester
 
 import (
 	"bytes"
+	ccontext "context"
```

Comment:
I think `context` is a drop in replacement for `x/net/context`, we probably shouldn't have both versions